index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Project</title>
+</head>
+<body>
+
+ <nav class="navbar">
+   <div class="logo">
+     My logo
```

Comment:
- I suggest you to turn your logo into an anchor tag, so that your visitor could get back to your home page when clicking it